### PR TITLE
[stable/mongodb-replicaset] fix harcoded busybox image

### PR DESCRIPTION
**Pull request by floriantraber**
Signed-off-by: Florian Traber <github@floriantraber.de>

#### What this PR does / why we need it:

Allow usage of a different busybox image

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

**Comment by steven-sheehy** [comment]
I've addressed the init timeout in another PR, since this one wasn't marked as fixing your ticket. Can you bump chart version, remove timeout and assign to unguiculus ?

**Comment by floriantraber** [comment]
Done.
/assign @unguiculus

File: stable/mongodb-replicaset/values.yaml [hunk 1]
```diff
@@ -24,6 +24,12 @@ installImage:
   tag: 0.6
   pullPolicy: IfNotPresent
 
+# Specs for the Docker image for the copyConfig init container
+copyConfigImage:
+  repository: "busybox"
+  tag: "latest"
```
Comment (unguiculus) [inline stable/mongodb-replicaset/values.yaml]: I know, this wasn't the case before, but we should use a stable version.

File: stable/mongodb-replicaset/values.yaml [hunk 2]
```diff
@@ -24,6 +24,12 @@ installImage:
...
+  pullPolicy: "Always"
```
Comment (unguiculus) [inline stable/mongodb-replicaset/values.yaml]: Use `IfNotPresent` as default.

**Review by unguiculus — CHANGES_REQUESTED** [review]

File: stable/mongodb-replicaset/values.yaml (hunk 1 above)
Comment (floriantraber) [reply stable/mongodb-replicaset/values.yaml]: Changed

File: stable/mongodb-replicaset/values.yaml (hunk 2 above)
Comment (floriantraber) [reply stable/mongodb-replicaset/values.yaml]: Changed

Comment (steven-sheehy) [reply stable/mongodb-replicaset/values.yaml]: Forgot README

File: stable/mongodb-replicaset/values.yaml (hunk 1 above)
Comment (steven-sheehy) [reply stable/mongodb-replicaset/values.yaml]: I think 1.29 would be safer as it's still considered stable and would still get bug fix releases

**Review by unguiculus — APPROVED** [review]

**Pull request merged**